src/SharpGLTF.Ext.3DTiles/Schema2/Ext.Features.cs

```diff
@@ -235,14 +235,6 @@ public MeshExtMeshFeatureID CreateFeatureID(IMeshFeatureIDInfo properties, Textu
                 instance.Label = properties.Label;
                 instance.Attribute = properties.Attribute;
                 instance.PropertyTableIndex = properties.PropertyTableIndex;
-
-                if (texture != null)
```

Comment:
must be added somewhere else?

Reply:
added again https://github.com/vpenades/SharpGLTF/pull/213/commits/93f3d5178e053d5962954ab99f5cc46d26085386